safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -362,6 +362,14 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 14062206, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 14062208, "1.3.0"),
     ],
+    EthereumNetwork.XDC: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 54712487, "1.3.0+L2"),
```

Comment:
Are you sure that the block number is the creation of the contract? 
https://xdc.blocksscan.io/txs/0xb6cf294d8c99756425bf8586f4ad4f7f15752d0d67c4aa95614b02efc6d94428#overview
I got this transaction hash from the PR that added this chain to safe-deployments.
https://github.com/safe-global/safe-deployments/pull/167